dashboard/v1.1/src/layouts/BaseLayout.tsx

```diff
@@ -64,10 +68,29 @@ const useStyles = makeStyles(theme => ({
   }
 }));
 
+const sunIcon = makeStyles(() => ({
+  root: {
+    position: 'absolute',
+    right: 5,
+    top: 3,
+    color: '#f39c12'
```

Comment:
Changing the icon colors to #fff makes it look minimalistic. Also this yellow grabs my attention away from the dashboard, which I'd not opt for.

Reply:
Okay sure.